content/post/wxs-reflection-002.md

```diff
@@ -0,0 +1,44 @@
+authors = ["Xavier Snelgrove"]
++++
+
+This week has been a less explicitly productive one for my research. However because of the auto-ethnographic quality to the work I'm doing, technically anything I do while inside the fellowship container is data on what it's like to be inside the fellowship container...
+
+It's hard to believe it's actually only been a week since our last meeting. Time has a funny way of bending and stretching when in these kinds of spaces. There's a currently fashionable ancient conception of time as having two faces: Chronos which is "clock time", the sort of linear sequential time in which one makes plans and calendar events, the sort of time that under our economic system one sells, the kind that we're always running out of. And then Kairos, which is the sort of atemporal circular multidimensional time of our direct experience, the time which expands when you're deep in the flow of conversation, the time which is contracted by the tyranny of a meeting agenda, the time which as you watch the sunset connects you to every other sunset you've watched.
+
+These two kinds of time are in a sort of tension with one another. Chronos has this quality of practical necessity: people with a poor handle on Chronos tend to be chronically[^1] late, have trouble delivering complex projects. Kairos of a humane and cosmic empathy. People with a poor handle on Kairos tend towards burnout, for themselves and their collaborators, tend to be rigid in their thinking and planning having trouble adapting to changing circumstances, and may not respond well to the information held in intuitions and the natural ebb and flow of their energy.
+
+There seems to be something powerful in being able to hold both of these times in balance. I'm struck with how this is seen in the language of the Catholic Church at its heyday, where there was a distinction made between its *temporal* and *atemporal* power; The temporal power of the church (read Chronos) was where it exerted control over mundane aspects of life, political and secular institutions. The atemporal power of the church (read Kairos) was its spiritual dimensions, understood to be somehow outside of the linear flow of time.
+
+So, too, it seems, the fellowship should play skillfully with these materials. I have [an are.na board](https://www.are.na/xavier-snelgrove/time-material)where I've been collecting images and writings on the materiality of time which I should continue to contemplate on as part of my work. Some questions that come up here:
+- How rigid should the temporal boundaries of the fellowship container be?
+- How early does one decide to be in the fellowship? How does the container reach out to fellows in that earlier time?
+- How rigid is the scheduling inside the container?
+- How crisp should be the boundary of "graduation" or "completion"?
+- How does the container reach out to alumni into the future?
+
+One might think of the Chronos cycle a fellow goes through as something like:
```

Comment:
There is something elegant about the alternating of "discrete" events and "zones" here - almost as if each event opens up onto a new prospective or a flow new, different energy